src/sites_toolbox.py

```diff
@@ -0,0 +1,64 @@
+import os
+import streamlit as st
+from datetime import datetime, timedelta, time, timezone
+from pvsite_datamodel.connection import DatabaseConnection
+from pvsite_datamodel.read import (
+    get_all_sites,
+    get_user_by_email,
+)
+from get_data import (
+    get_all_users, 
+    get_all_site_groups, 
+    update_user_site_group, 
+    )
+
+
+import plotly.graph_objects as go
+
+
+# get_user_details(): select user and show details for that user: Say how many sites the user has up the top
+# Add company name (also to db) 
+# Allow users to view and search on the sites’ user ID (I.e. what the users call the site)
+
+def get_user_details(session, email):
+  """Get the user details from the database"""
+  user_details = get_user_by_email(session=session,
+                                         email=email)
+  user_site_group = user_details.site_group.site_group_name
+  user_site_count = len(user_details.site_group.sites)
+  user_sites= [{"site_uuid": str(site.site_uuid), "client_site_id": str(site.client_site_id)} for site in user_details.site_group.sites]
```

Comment:
do we need to format this? run `blacks` maybe

Reply:
probably